alibi/explainers/integrated_gradients.py

```diff
@@ -112,13 +112,17 @@ def _select_target(preds: tf.Tensor,
     Selected predictions.
 
     """
+    if not isinstance(targets, tf.Tensor):
+        targets = tf.convert_to_tensor(targets)
```

Comment:
Is this necessary? If it is, then the `preds` type hint should change to accomodate other types. Otherwise conversion to `tf.Tensor` should be done outside this function.